tests/DataFixtures/ORM/resources/product_association_types.yml

```diff
@@ -0,0 +1,33 @@
+Sylius\Component\Product\Model\ProductAssociationTypeTranslation:
+    associationTypeTranslation1pl:
```

Comment:
We are really inconsistent in terms of defining references in fixtures... Something should be done with it.